gomint-server/src/main/java/io/gomint/server/network/packet/PacketStartGame.java

```diff
@@ -121,6 +122,7 @@ public void serialize( PacketBuffer buffer, int protocolID ) {
         buffer.writeBoolean( this.unknown1 );
         buffer.writeLLong( this.currentTick );
         buffer.writeSignedVarInt( this.enchantmentSeed );
+        buffer.writeBoolean( this.educationEdition );
```

Comment:
This is not part of the protocol in 1.5.2 or 1.6.0.8 where does this come from?

Reply:
@geNAZt He put it in the wrong place but https://github.com/pmmp/PocketMine-MP/blob/master/src/pocketmine/network/mcpe/protocol/StartGamePacket.php#L200

Reply:
Please remove the boolean, i will merge it then

Reply:
@geNAZt You do realise you have access to push commits to their fork if they enabled it on the PR?